test/config/va-remote-a.json

```diff
@@ -20,7 +20,7 @@
 		},
 		"grpc": {
 			"maxConnectionAge": "30s",
-			"address": ":9097",
+			"address": ":9397",
```

Comment:
Why does this change?

Reply:
This was not caught during review of https://github.com/letsencrypt/boulder/pull/7246/files#diff-2c48b7e6b30808c9b71be1ca3f2d6c1303af99b63814cdf642d0286c13e0efe7 and was causing failures during integration testing due to the RVA failure threshold being exceeded. Once I fixed this port to match [startservers.py](https://github.com/letsencrypt/boulder/blob/b3bb595075b3282f42a2ca8de2c9a071fbce08fc/test/startservers.py#L20), then tests began passing again.